src/background/extension.ts

```diff
@@ -67,36 +67,40 @@ export class Extension {
         }
 
         const {automation} = this.user.settings;
-
-        let timingInformation: TimeCheck = null;
+        let nextCheck: number;
         switch (automation) {
             case 'time':
-                timingInformation = isInTimeInterval(this.user.settings.time.activation, this.user.settings.time.deactivation);
+                this.isEnabledNow = isInTimeInterval(this.user.settings.time.activation, this.user.settings.time.deactivation);
+                nextCheck = nextIntervalTime(this.user.settings.time.activation, this.user.settings.time.deactivation);
                 break;
             case 'system':
                 if (isFirefox) {
                     // BUG: Firefox background page always matches initial color scheme.
-                    return this.wasLastColorSchemeDark == null
+                    this.isEnabledNow = this.wasLastColorSchemeDark == null
```

Comment:
Maybe let's rename `isEnabledNow`? Imagine someone sees `isEnabled` and `isEnabledNow` in code. He will not guess how do they differ.

Reply:
What should the new name be? Does `this.isEnabledCached` convey the way it is used?

Reply:
I think yes, `isEnabledCached` is fine.

Reply:
Done!